jest.config.js

```diff
@@ -1,15 +0,0 @@
-module.exports = {
```

Comment:
How come this file is being shown as deleted? I cloned your repo and the file is this there.